app/models/akka/RequesterBehavior.scala

```diff
@@ -34,9 +54,69 @@ trait RequesterBehavior { me: AbstractDSLinkActor =>
       processRequests(requests)
       requests.lastOption foreach (req => lastRid = req.rid)
     case e @ ResponseEnvelope(responses) =>
+
       log.debug("{}: received {}", ownId, e)
       cleanupStoredTargets(responses)
-      sendToEndpoint(e)
+
+      val(subscriptions, other) = responses.partition(isSubscription)
+
+      if(subscriptions.nonEmpty){
```

Comment:
I think I left that comment some place else: I'm trying to avoid using interpolation in logging statements wherever it may turn into a performance issue, and use `log.debug("...{}...{}...", arg1, arg2, ...)` instead

Reply:
replaced in all places i found it, thanks